src/main/scala/cromwell/engine/backend/package.scala

```diff
@@ -3,6 +3,11 @@ package cromwell.engine
 import scala.concurrent.{ExecutionContext, Future}
 
 package object backend {
+  // Ordered by shards, and then ordered by attempts
+  type AttemptedCallLogs = Seq[Seq[CallLogs]]
```

Comment:
Scala collections-- stay weird. _Why tf are Map and Set immutable by default, but not the other collections??_